lib/avo/fields/belongs_to_field.rb

```diff
@@ -259,8 +259,8 @@ def get_model
         nil
       end
 
-      def name
-        return polymorphic_as.to_s.humanize if polymorphic_as.present? && view.index?
```

Comment:
Removing `&& view.index?` is ok? I mean it does seem to have it's place...

Reply:
I think so, why we would apply the polymorphic_as default name only on index? That means that on other views would fallback to 

```ruby
def default_name
    @id.to_s.humanize(keep_id_suffix: true)
  end
```